adapter/internal/api/apis_impl.go

```diff
@@ -56,6 +56,7 @@ func extractAPIProject(payload []byte) (apiProject model.ProjectAPI, err error)
 	// TODO: (VirajSalaka) this won't support for distributed openAPI definition
 	apiProject.UpstreamCerts = make(map[string][]byte)
 	apiProject.EndpointCerts = make(map[string]string)
+	apiProject.Policies = make(map[string]model.PolicyContainer)
```

Comment:
Let's add this here too https://github.com/wso2/product-microgateway/blob/348cad1cd2dbedf32672234c4cdb1990462dfd3e/adapter/internal/api/apis_impl.go#L95